backend/trafficrules/src/main/java/com/coffeejug/trafficrules/service/UserService.java

```diff
@@ -40,4 +59,23 @@ private UUID parseUUID(String uuid) {
             throw new BadRequestException("Invalid UUID " + uuid);
         }
     }
+
+    public long count() {
+        return userRepository.count();
+    }
+
+    public long countAllByRegisteredAfter(LocalDateTime localDateTime) {
+        if (localDateTime == null) {
+            return count();
+        }
+        return userRepository.countAllByRegisteredAfter(localDateTime);

```

Comment:
Since we are searching for all registered users, can't we just use `userRepository.count()` here?

Reply:
We use it also in statistic (users registered from application start).